accounts/forms.py

```diff
@@ -1,9 +1,23 @@
 from django import forms
 from .models import User
+from django.forms import ValidationError
 
 
 class RegistrationForm(forms.ModelForm):
-
     class Meta:
         model = User
-        fields = ['email', ]
+        fields = ['display_name', 'full_name', 'email']
+        widgets = {
+            'email': forms.TextInput(attrs={'class': 'uk-form-width-medium', 'required': 'required'}),
```

Comment:
Don't sweat it... I can make widget files from the templates you've made already...